tests/test_init.py

```diff
@@ -92,6 +92,10 @@ def test_init_vertical():
 
     Simul(params)
 
+    params.oper.sfd_activation = 1.0
```

Comment:
There should be a specific test for the SFD method. You can keep this test unmodified.

Reply:
I changed it to `enable_sfd`. By the way, I didn't find a good way to communicate `bool` to `real` in `fortran`.